coding-concepts.js

```diff
@@ -10,34 +10,34 @@
 const cohort = "Bravo 2023"
 // console.log(cohort.split(""))
 
-// a) Your answer:
-// b) Verify and explain:
+// a) Your answer: "B" "r" "a" "v" "o" "2" "0" "2" "3"
+// b) Verify and explain: The split method had quotation marks without any spaces as the argument which means it will return each individual character in quotation marks.
```

Comment:
Yes! And it's important to note that the space is included and the output is an array.